router/batchrouter/asyncdestinationmanager/bing-ads/util.go

```diff
@@ -392,18 +437,28 @@ func GetFailedReasons(clientIDErrors map[string]map[string]struct{}) map[string]
 
 // filtering out failed jobIds from the total array of jobIds
 // in order to get jobIds of the successful jobs
-func GetSuccessKeys(failedEventList, initialEventList []int64) []int64 {
+func GetSuccessJobIDs(failedEventList, initialEventList []int64) []int64 {
 	successfulEvents := make([]int64, 0)
 
-	lookup := make(map[int64]bool)
+	failedEventsMap := make(map[int64]bool)
 	for _, element := range failedEventList {
-		lookup[element] = true
+		failedEventsMap[element] = true
 	}
 
 	for _, element := range initialEventList {
-		if !lookup[element] {
+		if !failedEventsMap[element] {
 			successfulEvents = append(successfulEvents, element)
 		}
 	}
 	return successfulEvents
 }
+
+// If any one importId is in progress, then the overall completion status is false
+func generateCompletionStatus(array []bool) bool {
```

Comment:
return !lo.Contains(array, false), this function itself may not be needed now

Reply:
using lo.Difference